Panic runtime with wolfictl advisory discover command

**Description**

Inside a wolfi-sdk container  when running the following command 
`GITHUB_TOKEN=github_pat_xxxx wolfictl advisory discover -d wolfi-dir/ -a advisories-dir/ -p spark-3.4 -r https://my_package_url`
I have the following error :
```go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xab95dc]
goroutine 20 [running]:
  github.com/wolfi-dev/wolfictl/pkg/advisory.DiscoverOptions.filterMatchesForPackage({{0x400108e520, 0x1, 0x1}, 0x400010f080, 0x400010ee80, {0xffffc2f99dfd, 0x5a}, {0x40010580c0, 0x2, 0x2}, ...}, ...)
  github.com/wolfi-dev/wolfictl/pkg/advisory/discover.go:120 +0x25c
  github.com/wolfi-dev/wolfictl/pkg/advisory.DiscoverOptions.discoverMatchesForPackage({{0x400108e520, 0x1, 0x1}, 0x400010f080, 0x400010ee80, {0xffffc2f99dfd, 0x5a}, {0x40010580c0, 0x2, 0x2}, ...}, ...)
  github.com/wolfi-dev/wolfictl/pkg/advisory/discover.go:98 +0x194
  github.com/wolfi-dev/wolfictl/pkg/advisory.Discover({0x39a4110, 0x400075c5f0}, {{0x400108e520, 0x1, 0x1}, 0x400010f080, 0x400010ee80, {0xffffc2f99dfd, 0x5a}, {0x40010580c0, ...}, ...})
  github.com/wolfi-dev/wolfictl/pkg/advisory/discover.go:76 +0x2b4
  github.com/wolfi-dev/wolfictl/pkg/cli.cmdAdvisoryDiscover.func1.2()
  github.com/wolfi-dev/wolfictl/pkg/cli/advisory_discover.go:96 +0x244
  golang.org/x/sync/errgroup.(*Group).Go.func1()
  golang.org/x/sync@v0.7.0/errgroup/errgroup.go:78 +0x58
created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 1
  golang.org/x/sync@v0.7.0/errgroup/errgroup.go:75 +0x98
```
Could you help me on this ?

For information the wolfictl version i use : 
```bash
> wolfictl version
 __        __   ___    _       _____   ___    ____   _____   _
 \ \      / /  / _ \  | |     |  ___| |_ _|  / ___| |_   _| | |
  \ \ /\ / /  | | | | | |     | |_     | |  | |       | |   | |
   \ V  V /   | |_| | | |___  |  _|    | |  | |___    | |   | |___
    \_/\_/     \___/  |_____| |_|     |___|  \____|   |_|   |_____|
wolfictl: A CLI helper for developing Wolfi

GitVersion:    v0.18.0-18-g5f9a2fa
GitCommit:     5f9a2fa42bf3d3ff75bc9ffcb6dd3180e4dea208
GitTreeState:  clean
BuildDate:     '1970-01-01T00:00:00Z'
GoVersion:     go1.22.4
Compiler:      gc
Platform:      linux/arm64
```
The spark-3.4.yaml package file exist and the package was successfully created.

--- a/pkg/advisory/discover.go
+++ b/pkg/advisory/discover.go
@@ -95,7 +95,10 @@ func (opts DiscoverOptions) discoverMatchesForPackage(ctx context.Context, pkg s
 		opts.VulnEvents <- vuln.EventPackageMatchingError{Package: pkg, Err: err}
 	}
 
-	matches = opts.filterMatchesForPackage(pkg, matches)
+	matches, err = opts.filterMatchesForPackage(pkg, matches)
+	if err != nil {
+		return fmt.Errorf("filtering vulnerability matches: %w", err)
+	}
 
 	opts.VulnEvents <- vuln.EventPackageMatchingFinished{Package: pkg, Matches: matches}
 
@@ -115,8 +118,11 @@ func (opts DiscoverOptions) discoverMatchesForPackage(ctx context.Context, pkg s
 	return nil
 }
 
-func (opts DiscoverOptions) filterMatchesForPackage(pkg string, matches []vuln.Match) []vuln.Match {
-	buildCfgEntry, _ := opts.BuildCfgs.Select().WhereName(pkg).First() //nolint:errcheck
+func (opts DiscoverOptions) filterMatchesForPackage(pkg string, matches []vuln.Match) ([]vuln.Match, error) {
+	buildCfgEntry, err := opts.BuildCfgs.Select().WhereName(pkg).First()
+	if err != nil {
+		return nil, fmt.Errorf("finding package %q among build configurations: %w", pkg, err)
+	}
 	buildCfg := buildCfgEntry.Configuration()
 
 	var filteredMatches []vuln.Match
@@ -150,7 +156,7 @@ func (opts DiscoverOptions) filterMatchesForPackage(pkg string, matches []vuln.M
 		filteredMatches = append(filteredMatches, match)
 	}
 
-	return filteredMatches
+	return filteredMatches, nil
 }
 
 func advisoryEventForNewDiscovery(match vuln.Match) v2.Event {

